docs/cookbook.rst

```diff
@@ -312,7 +317,7 @@ If you want to run the full packaging flow where feature branches build unmanage
 
 * CUMULUSCI_ORG_packaging: The output from `cci org info packaging`, assuming you've already connected your packaging org to your local toolbelt.
 * CUMULUSCI_ORG_beta: The output from `cci org info beta`, assuming you've already connected your beta org to your local toolbelt.
-* CUMULUSCI_SERVICE_github: The output from `cci project show_github`, assuming you've already configured github locally via `cci project connect_github` 
+* CUMULUSCI_SERVICE_github: The output from `cci project show_github`, assuming you've already configured github locally via `cci project connect_github`
```

Comment:
Now that I think about it, is this line still valid? We can address in another PR.

Reply:
Logged in #1173.